local-volume/README.md

```diff
@@ -148,7 +148,14 @@ spec:
   local:
     path: /mnt/disks/ssd1
 ```
-
+Please replace the following elements to reflect your configuration:
+  * "my-node" with the name of kubernetes node which is hosting this
+    local storage disk
+  * "5Gi" with required size of storage volume
```

Comment:
This should be the actual capacity of the volume.  The user's request for size is in the PVC.